src/foam/u2/Element.js

```diff
@@ -865,7 +865,13 @@ foam.CLASS({
       class: 'Object',
       name: 'id',
       transient: true,
-      factory: function() { return this.NEXT_ID(); }
+      // factory: function() { return this.NEXT_ID(); }
+      factory: function() {
+        var id = this.NEXT_ID();
+        if ( ! window.idToE ) window.idToE = {};
+        window.idToE[id] = this;
```

Comment:
This is going to prevent views from being garbage collected. Did you mean for this to be here? If you store the view in its el() then GC won't be a problem and then you could do getElementById(...).u2el (or something).

Reply:
Left this by accident. I'll make a PR to store this on the element as you suggested but I'm not sure where it's safe to do so - is it possible for `el` to change after `load` is called? (I'm thinking maybe `load` is where I should do that)